src/index.html

```diff
@@ -154,6 +164,49 @@ <h2 class="card__title"></h2>
         </div>
       </li>
     </template>
+
+    <div class="modal" id="remove-card-popup">
+      <div class="modal__container">
+        <button
+          aria-label="Close modal"
+          type="button"
+          class="modal__close"
+        ></button>
+        <h3 class="modal__heading">Are you sure?</h3>
+        <form class="modal__form" id="remove-card-form" novalidate>
+          <button type="submit" class="button modal__button">Yes</button>
+        </form>
+      </div>
+    </div>
+
+    <!-- Avatar Modal -->
+
+    <div class="modal" id="avatar-modal">
+      <div class="modal__container">
+        <button
+          aria-label="Close modal"
+          type="button"
+          class="modal__close"
+        ></button>
+        <h3 class="modal__heading">Are you sure?</h3>
```

Comment:
* Please, add the correct **texts** for the `avatar` modal https://snipboard.io/gr0dLw.jpg